x-pack/plugins/ml/public/util/theme_colors.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import chrome from '../../../../../src/legacy/ui/public/chrome';
+const IS_DARK_THEME = chrome.getUiSettingsClient().get('theme:darkMode');
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
```

Comment:
I'm not going to worry about it now, but this an instance where we'll run into trouble when we start having custom theming.

Reply:
Yes. There will eventually be a theme provider. But at least if we use this it'll all be in one place so we just update the one file. That's why I like `euiColorForTheme` as a name? Guess it doesn't matter too much.

Reply:
Yeah I was just thinking on the consumer side, they don't need to care about the `theme` part of it, they just care that they're getting the right color. So something simple like `euiColor` would be easier for them to remember. Though, maybe it shouldn't be prefixed with `eui` since it could be a custom theme. So maybe just `getColor` though that's probably too generic... dunno maybe more of a kibana service so `kbnGetColor`?